### PR TITLE
support live activities

### PR DESCRIPTION
**Pull request by JO0st**

### CODE REVIEW

File: src/main/java/com/eatthepath/pushy/console/PushyConsoleApplication.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public class PushyConsoleApplication extends Application {
     private PushyConsoleController pushyConsoleController;
 
     // Based heavily upon https://softwarei18n.org/using-unicode-in-java-resource-bundles-6220776b6099
-    static final ResourceBundle RESOURCE_BUNDLE = ResourceBundle.getBundle("com/eatthepath/pushy/console/pushy-console");
+    static final ResourceBundle RESOURCE_BUNDLE = ResourceBundle.getBundle("com/eatthepath/pushy/console/pushy-console_en");
```
Comment (jchambers) [inline src/main/java/com/eatthepath/pushy/console/PushyConsoleApplication.java:44]: Is this change necessary to support live activities? If not, could we save this for a separate pull request?

**Review by jchambers — CHANGES_REQUESTED** [review]
Thank you for the contribution!

File: src/main/java/com/eatthepath/pushy/console/PushyConsoleApplication.java (hunk 1 above)
Comment (bhaskarmelkani) [reply src/main/java/com/eatthepath/pushy/console/PushyConsoleApplication.java:44]: I don't think its necessary :)

Comment (JO0st) [reply src/main/java/com/eatthepath/pushy/console/PushyConsoleApplication.java:44]: That's indeed not necessary for the live activities. I needed it to get the console working and accidently added it to the commit. I will take it out.

**Review by jchambers — APPROVED** [review]
Thank you again for this contribution and for your patience!
